pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_config_repo.adoc

```diff
@@ -0,0 +1,29 @@
+[id="configuring-a-repository-as-a-content-source-for-pantheon_{context}"]
+
+= Configuring a repository as a content source for Pantheon
+
+.Procedure
+
+. In a text editor, paste the content of the sample configuration file in xref:pantheon-yaml-file_{context}[] into a new text file. Name the file [filename]`pantheon2.yml`. This file is the Pantheon configuration file for your repository.
+
+. Replace the default value for ‘server’ with the URL you receive from a Pantheon administrator. and ‘repository’ with U:
```

Comment:
`and ‘repository’ with U` seems like this is a text artifact that should be removed?

Reply:
Yep. I missed that.

Reply:
Fixed.